docker/native-linux-image.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
The major obstacle to not having a wrapper for this is `linux/arm`, which is `armv7`, but `linux/arm/v6` is `arm`, so we can't just use `${TARGETARCH}${TARGETVARIANT}`.